annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -92,11 +101,26 @@ fun main(argv: Array<String>) {
         null
     }
 
+    // Build params for extracting the annotation statistics
     val extractAnnotationStatsParamsBuilder = Parameters.builder()
     extractAnnotationStatsParamsBuilder.set("exportedAnnotationRoot", "$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
     extractAnnotationStatsParamsBuilder.set("statisticsDirectory", params.getString("statisticsDirectory"))
     val extractAnnotationStatsParams = extractAnnotationStatsParamsBuilder.build()
 
+    // Assemble params for converting the JSON to FlexNLP documents
+    val inputAnnotationJsonDir = File("$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
+    val annotatedFlexNLPOutputDir = File(params.getString("flexnlpDocumentDirectory"))
+    val ingesterParametersDir = File(params.getString("ingesterParametersDirectory"))
+    val ingesterParametersPath = "$ingesterParametersDir/curated_training_ingester_params.yaml"
```

Comment:
If you get `ingesterParametersDir` as a `Path` you can do `ingesterParametersDir.resolve("curated_Training_ingester_parans.yaml")`